src/pages/resource/ResourceDescriptionForm.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { Field, Form, Formik } from 'formik';
+import { useTranslation } from 'react-i18next';
+import * as Yup from 'yup';
+import DateFnsUtils from '@date-io/date-fns';
+import { Select, TextField } from 'formik-material-ui';
+import { Button, MenuItem } from '@material-ui/core';
+import '../../styles/pages/resource/resource-description.scss';
+import { defaultLanguage, languages } from '../../translations/i18n';
+import publications from '../../utils/testfiles/projects_random_generated.json';
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import FormikDatePicker from './FormikDatePicker';
+import { CLEAR_PUBLICATION_ERRORS, PUBLICATION_ERROR } from '../../redux/actions/validationActions';
+
+export interface ResourceDescriptionFormProps {
+  dispatch: any;
+}
+
+const ResourceDescriptionForm: React.FC<ResourceDescriptionFormProps> = ({ dispatch }) => {
+  const { t } = useTranslation();
+
+  const resourceSchema = Yup.object().shape({
+    title: Yup.string().required(t('Required field')),
+    abstract: Yup.string().required(t('Required field')),
+    description: Yup.string().required(t('Required field')),
+  });
+
+  const handleValidation = (values: any) => {
```

Comment:
Kunne vært greit med en definert type for `values` også?

Reply:
undersøkes grundigere ved en senere anledning ...